jsx/Form.js

```diff
@@ -1389,23 +1454,21 @@ FileElement.defaultProps = {
  *    label={note}
  * />
  * ```
+ *
+ * @param {object} props
+ * @return {jsx}
  */
-class StaticElement extends Component {
-  constructor(props) {
-    super(props);
-  }
-  render() {
-    return (
-      <div className="row form-group">
-        <label className="col-sm-3 control-label">
-          {this.props.label}
-        </label>
-        <div className="col-sm-9">
-          <p className="form-control-static">{this.props.text}</p>
-        </div>
+function StaticElement(props) {
+  return (
+    <div>
+      <label className="control-label">
+        {props.label}
+      </label>
+      <div>
+        <p className="form-control-static">{props.text}</p>
       </div>
-    );
-  }
+    </div>
+  );
```

Comment:
not in **24.1**

_24.1 seems to have made some updates to this Element, and I think they will likely work given what code is here_

to adopt **24.1**